docroot/themes/contrib/civictheme/includes/table.inc

```diff
@@ -0,0 +1,72 @@
+<?php
+
+/**
+ * @file
+ * Table theme alterations.
+ */
+
+define('CIVICTHEME_OPT_OUT', 'CivicThemeOptOut');
+
+/**
+ * Implements template_preprocess_table().
+ */
+function civictheme_preprocess_table(&$variables) {
+  // Header.
+  if (!empty($variables['header'])) {
+    $rows = NULL;
+    foreach ($variables['header'] as $row) {
+      $rows[] = $row['content'];
+    }
+    $variables['header'] = $rows;
+  }
+
+  // Rows, Footer.
+  foreach (['rows', 'footer'] as $section) {
+    $rows = NULL;
+
+    if (!empty($variables[$section])) {
+      foreach ($variables[$section] as $row_key => $row) {
+        foreach ($row['cells'] as $cell) {
+          $rows[$row_key][] = $cell['content'];
+        }
+      }
+    }
+
+    $variables[$section] = $rows;
+  }
+}
+
+/**
+ * Implements template_preprocess_HOOK() for views_view_table.
+ */
+function civictheme_preprocess_views_view_table(&$variables) {
+  $view = $variables['view'];
+  $variables['opt_civictheme'] = TRUE;
+
+  if ($view->storage->get('tag') !== CIVICTHEME_OPT_OUT) {
```

Comment:
can we please get multiple views tags and check that at least one of them has the optout

Reply:
@AlexSkrypnyk Added support to check if multiple